utils/kvrocks2redis/kvrocks2redis.conf

```diff
@@ -17,6 +17,11 @@ daemonize no
 # Default: ./data
 data-dir ./data
 
+# The Secondary Instance directory.
+#
+#  Default: ./secondary
+secondary-dir ./secondary
+
```

Comment:
I'm wondering if we could just use `data-dir` here instead of introducing a `secondary-dir`?

Reply:
I mean, maybe we can make secondary instance the default way.

Reply:
I may have not understood your meaning correctly, but are you referring to configuring the secondary dir similar to `db_dir = data_dir + "/db"`, without introducing a `secondary-dir`?

Reply:
Currently we use `OpenForReadOnly` in kvrocks2redis, we can change it to `OpenAsSecondary` and then we will be able to use `TryCatchUpWithPrimary` to capture incremental data rather than network connections.

That's the intention of #1832.

Reply:
oh, this pr was created for that issue. Then I should be able to remove the support for the readonly mode in the `storage::Open`.

Reply:
I think it is optional to clean up Storage::Open, but I suggest you to clean the code for original logic and useless network connections in kvrocks2redis.

Reply:
Ok, I understand what you mean now